packages/graph/hash_graph/lib/graph/src/store/postgres/query/compile.rs

```diff
@@ -174,7 +174,7 @@ impl<'c, 'p: 'c, T: PostgresQueryRecord + 'static> SelectCompiler<'c, 'p, T> {
     // Warning: This adds a CTE to the statement, which is overwriting the `type_ids` table. When
     //          more CTEs are needed, a test should be added to cover both CTEs in one statement to
     //          ensure compatibility
-    fn compile_latest_version_filter(
+    fn compile_latest_ontology_version_filter(
         &mut self,
         path: &T::Path<'p>,
         operator: EqualityOperator,
```

Comment:
(comment for line 195-200) can't we get rid of this part of the match?

Reply:
Thank you, yes, that is not needed anymore. I fixed this in 3835fae50e3f353f511ef3d0dba3e868a07324ee